README.md

```diff
@@ -122,16 +122,14 @@ You will need to install the following required packages on the build system:
 
 * **autoconf**
 * **pkg-config**
-* **libndctl-devel** (v60.1 or later)<sup>1</sup>
-* **libdaxctl-devel** (v60.1 or later)
+* **libndctl-devel** (v63 or later)
+* **libdaxctl-devel** (v63 or later)
 
 The following packages are required only by selected PMDK components
 or features:
 
 * **libfabric** (v1.4.2 or later) -- required by **librpmem**
 
-><sup>1</sup> PMDK requires libndctl v63 or newer (to enable RAS features) by default. If you have such libndctl version but you don't want to depend on this API please build PMDK with NDCTL_USE_V63_API=n. For more information please check this issue pmem/issues#1039
```

Comment:
This is still relevant, but the variable changed, right?

Reply:
No, with this PR PMDK works only with libndctl >= 63.

Reply:
`NDCTL_ENABLED` variable doesn't work anymore?
As is, this text makes sense even after this change if I understand this correctly.

Reply:
resolved offline.